css/main.css

```diff
@@ -11,6 +11,19 @@ body {
 	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
 }
 
+body.flash {
```

Comment:
Let's make this class name more semantic (something like `flash-food` maybe?) We may have other flashes (`flash-win`, `flash-lose`, different colors?)